cc/crypto/hpke/jni/README.md

```diff
@@ -0,0 +1,5 @@
+# Oak HPKE crypto JNI
+
+JNI headers generated using:
+
+`javac -h cc/crypto/hpke/jni java/src/main/java/com/google/oak/crypto/hpke/*.java java/src/main/java/com/google/oak/util/Result.java`
```

Comment:
Can this command be run from our [dev Docker](https://github.com/project-oak/oak/blob/main/Dockerfile)?
Or it requires adding something there